addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1149,7 +1156,7 @@
             we-button {
                 display: flex;
                 justify-content: center;
-                flex: 0 1 percentage(1/4);
+                flex: 1 1 percentage(1/4);
```

Comment:
the button and the label should be spaced-out 

![image](https://user-images.githubusercontent.com/8558351/234522714-430d079d-fc5f-4ffd-a215-c1ef4f478ed9.png)


**note: not sure if it's the right line !**